capstone/src/main/webapp/group.html

```diff
@@ -17,6 +17,9 @@
     <script src="scripts/poll.js"></script>
     <script src="scripts/members.js"></script>
     <script src="https://unpkg.com/ionicons@5.1.2/dist/ionicons.js"></script>
+    <script>
+      window.onload = getGroupId();
+ </script>
```

Comment:
Could this be added to the init in posts.js instead? Or perhaps create a new group.js file and move the init from posts.js into there?